requirements.txt

```diff
@@ -10,3 +10,7 @@ zarr==2.2.0
 pandas==0.23.4
```

Comment:
In favor of minimizing deployed package sizes, I've stopped using this top level requirements.txt and specifying requirements specific to each package. Moving to appropriate requirements files